### PR TITLE
Update using_letsencrypt.adoc

### PR DESCRIPTION
**Pull request by ChrisEdS**
Add missing `sh` to the `sudo /etc/letsencrypt/*` commands

### CODE REVIEW

**Review by voroyam — APPROVED** [review]
because it was tested by @ChrisEdS I approve this PR

**Comment by mmattel** [comment]
pls add a note for backporting and do post merging

File: modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc [hunk 1]
```diff
@@ -304,7 +304,7 @@ which can be run as follows:
 
 [source,console]
 ----
-sudo /etc/letsencrypt/list.sh
+sudo sh /etc/letsencrypt/list.sh
```
Comment (phil-davis) [inline modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc:307]: The `sh` won't hurt, but this should run fine without `sh` - what goes wrong? Why do we need to add `sh` here?

Comment (mmattel) [reply modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc:307]: sh is not needed, also see the other PR for the same thing

Comment (phil-davis) [reply modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc:307]: @ChrisEdS please respond and let us know why you think that `sh` is needed here.

Comment (ChrisEdS) [reply modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc:307]: I tested during a deployment and I had to add the `sh`. Maybe I did something wrong.

Comment (mmattel) [reply modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc:307]: I can prove you that `sh` is not necessary. I am using LE sucessfully including scripts I created for myself.

**Pull request closed without merging**